R/convert_year_types.R

```diff
@@ -0,0 +1,46 @@
+#' Convert year types - Financial year form to the alternate form
+#' 
+#' Converts year type from the financial year form '1718' to the year form '2017'.
+#'
+#' @param fyyear vector of financial years in the form '1718'
+#'
+#' @return a vector of years in the alternate form '2017'
+#' @export
+#'
+#' @examples
+#' fyyears <- c("1718", "1819")
+#' convert_fyyear_to_year(fyyears)
+#' [1] "2017" "2018"
```

Comment:
Remove this line please, the examples are run and the output is generated when you run `devtools::document`. So this isn't needed (and possibly breaks the documentation output)